packages/styling/src/styles/IconCodes.ts

```diff
@@ -2951,3 +2951,11 @@ export const IconCodes = {
   eventDeclined: '\uF425',
 
 };
+
+/**
+ * Get IconCode by IconName.
+*/
+export function getIconCodeByIconName(iconName: string): string {
```

Comment:
so i have this same thing in my PR, which really changes icons and moves this to a separate icons package. I'd hate for you to take a dependency on this only to have it removed in 5.0. Can we not make this change here?